libcomps/src/python/tests/comps/comps-f21.xml

```diff
@@ -20084,7 +20084,7 @@
     <match name="tesseract" install="tesseract-langpack-%s"/>
     <match name="tkgate" install="tkgate-%s"/>
   </langpacks>
-  <blacklist>
```

Comment:
@j-mracek I don't think the format of the metadata can be changed at will like this, can it?

Reply:
I would think that, at the very least, it needs to keep working with older metadata, and be tested to do so?

Reply:
I don't think we can assume that all clients will all know how to parse the new metadata?  Is comps metadata used in RHEL or CentOS at all?

Reply:
Hello, I can make a pr to revert the metadata change

Reply:
so, to revert the changes this PR would do it. I think backward compatibility is a good point
https://github.com/rpm-software-management/libcomps/pull/79